girder/__init__.py

```diff
@@ -74,6 +76,28 @@ def format(self, record, *args, **kwargs):
         return super(LogFormatter, self).format(record, *args, **kwargs)
 
 
+class StreamToLogger:
+    """
+    Redirect a file-like stream to a logger.
+    """
+    def __init__(self, stream, logger, level):
+        self.stream = stream
+        self.logger = logger
+        self.level = level
+        self.logger._girderLogHandlerOutput = False
+        for key in dir(stream):
+            if key != 'write' and not key.startswith('_') and callable(getattr(stream, key)):
+                setattr(self, key, getattr(stream, key))
```

Comment:
@manthey Your explanation here was really useful in understanding what wasn't obvious to me otherwise.  Can you add this info as a comment in the code?